nexus/catalog/migrations/V40__maintenance_flows.sql

```diff
@@ -0,0 +1,15 @@
+CREATE SCHEMA IF NOT EXISTS maintenance;
+
+CREATE TABLE IF NOT EXISTS maintenance.maintenance_flows
+(
+    id              SERIAL PRIMARY KEY,
+    flow_id         serial    NOT NULL,
```

Comment:
```suggestion
    flow_id         bigint    NOT NULL,
```
why is flow_id serial? type in code is 64bit, not 32bit